src/views/layout/RouterPage.vue

```diff
@@ -1,6 +1,23 @@
 <template>
-    <div class="page-wrapper">
-        <router-view />
+    <div class="wrap">
+        <div style="display: flex; width: 100%; height: 100%;">
+            <div class="general_view" style="width: 100%;">
+                <div class="top_window level">
+                    <div class="level-left"></div>
+                    <div class="level-right"></div>
+                </div>
+                <transition name="fade" mode="out-in">
+                    <div class="main-outer-container">
+                        <div class="page-wrapper">
+                            <router-view />
+                        </div>
+                    </div>
+                </transition>
+                <div class="footer">
+                    <span class="footer-phrase">Powered by <b>NEM Technology.</b></span>
```

Comment:
copyright?

Also NEM Technology doesn't seem right to me, NEM Software perhaps

Reply:
RouterPage.vue is basically copy of PageLayout.vue as unprotected(without menu), that's why we have it here too.
![image](https://user-images.githubusercontent.com/6256269/108068272-c71e8b00-7059-11eb-9911-015a6d69e036.png)

Reply:
Hmm, I think that's wrong, Should be changed to NEM Software, we don't have a NEM Technology entity ?

Reply:
fixed => Copyright © 2021 NEM Software